diagnostics-app/src/app/app-routing.module.ts

```diff
@@ -11,11 +11,13 @@ import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { DashboardComponent } from './telemetry/dashboard/dashboard.component';
 import { BatteryComponent } from './diagnostics/battery/battery.component';
+import { EventComponent } from './event/event.component';
 
 const routes: Routes = [
   { path: '', redirectTo: '/telemetry', pathMatch: 'full' },
   { path: 'telemetry', component: DashboardComponent },
-  { path: 'diagnostics/battery', component: BatteryComponent },
+  { path: 'diagnostics', component: BatteryComponent },
```

Comment:
I know we are not at the diagnostics card/page yet, but we might want to think about how we name the components.

Maybe we can discuss how we want to name the individual components in telemetry, diagnostics and event page.

Possible option might be naming the component just `TelemetryComponent`, `DiagnosticsComponent` and `EventComponent`.

If you want to specify its a dashboard page of some sort, maybe event `TelemetryDashboardComponent`, `EventDashboardComponent`, etc.

Or another naming can be `TelemetryPageComponent`, etc.

I think we would want consistent naming between the three pages, but feel free to choose how you want to name them.

Reply:
I think I will name them "TelemetryDashboardComponent", "DiagnosticsDashboardComponent", and "EventsDashboardComponent". And the changes will be made in another PR since it is not related to this one.

Reply:
Sure. SGTM.